scripts/solution_view.py

```diff
@@ -146,13 +146,14 @@ def update_table(self):
     self._table_list = self.table_spp.items()
 
   def pos_llh_callback(self, data):
-    soln = sbp_messages.PosLLH(data)
+    soln = MsgPosLLH(data)
```

Comment:
... so the serial link callback here is using the full SBP message? If so, this will be clearer with a different name (like `sbp_msg` or something). 

Also, are you importing in the serial driver from libsbp?